### PR TITLE
Feature Statistics page and module

### PR DESCRIPTION
**Pull request by em0nti**
Implement base markup and styling for the Statistics page and module. Please review the code.

### CODE REVIEW

File: src/modules/Statistics/Statistics.module.css [hunk 1]
```diff
@@ -1,46 +1,140 @@
...
+
+.stat__chartWrapper {
+  height: rem(414);
+
+  padding-top: rem(40);
+  padding-bottom: rem(40);
+  padding-left: rem(14);
+  padding-right: rem(14);
+
+  border: rem(1) solid var(--mantine-color-blue-1);
```
Comment (SalOne22) [inline src/modules/Statistics/Statistics.module.css:70]: Please add dark style with `light-dark()` function

File: src/modules/Statistics/Statistics.module.css [hunk 2]
```diff
@@ -1,46 +1,140 @@
...
+
+.stat__legendItem {
+  display: flex;
+  align-items: center;
+  gap: rem(8);
+
+  font-family: Poppins;
+  font-size: rem(16);
+  line-height: rem(18);
+  color: var(--mantine-color-dark-6);
```
Comment (SalOne22) [inline src/modules/Statistics/Statistics.module.css:110]: Here as well, add dark styles

File: src/modules/Statistics/Statistics.module.css [hunk 3]
```diff
@@ -1,46 +1,140 @@
...
+}
+
+.stat__legendDot_color_blue {
+  background-color: var(--mantine-color-blue-4);
+}
+
+.stat__chartTitle {
+  margin-bottom: rem(12);
+
+  color: var(--mantine-color-dark-6);
```
Comment (SalOne22) [inline src/modules/Statistics/Statistics.module.css:130]: Write dark style

**Review by SalOne22 — CHANGES_REQUESTED** [review]
All good, but needs improvements in dark styles

**Comment by SalOne22** [comment]
And please run prettier on your files

File: src/modules/Statistics/Statistics.module.css (hunk 2 above)
Comment (em0nti) [reply src/modules/Statistics/Statistics.module.css:110]: Done

File: src/modules/Statistics/Statistics.module.css (hunk 1 above)
Comment (em0nti) [reply src/modules/Statistics/Statistics.module.css:70]: Done

File: src/modules/Statistics/Statistics.module.css (hunk 3 above)
Comment (em0nti) [reply src/modules/Statistics/Statistics.module.css:130]: Done

**Review by SalOne22 — APPROVED** [review]

**Comment by em0nti** [comment]
The `.prettierrc` configuration file in the project directory currently specifies only one rule. 
```json
{
  "singleQuote": true,
}

I've been using my own Prettier settings locally and this is most likely the problem

**Pull request merged**